scalene/scalene_profiler.py

```diff
@@ -224,9 +235,16 @@ def get_original_lock() -> threading.Lock:
     __program_being_profiled = Filename("")
 
     # Is the thread sleeping? (We use this to properly attribute CPU time.)
-    __is_thread_sleeping: Dict[int, bool] = defaultdict(bool)  # False by default
+    __is_thread_sleeping: Dict[int, bool] = defaultdict(
+        bool
+    )  # False by default
     __child_pids: Set[int] = set()
 
+    # Signal queues for CPU timers, allocations, and memcpy
+    __cpu_sigq: ScaleneSigQueue[Any]
+    __alloc_sigq: ScaleneSigQueue[Any]
+    __memcpy_sigq: ScaleneSigQueue[Any]
+
```

Comment:
Maybe set these to `None` here?

Reply:
I didn't do that because then they have to be `Optional` types, and anyway they get initialized by `__init__`.